includes/class-wc-calypso-bridge-action-header.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Removes the back links and adds the action_header on settings pages.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.0
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Action Header
+ */
+class WC_Calypso_Bridge_Action_Header {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Action_Header instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor
+	 */
+	private function __construct() {
+		add_action( 'wp_after_admin_bar_render', array( $this, 'render' ) );
+		add_action( 'wp_loaded', array( $this, 'remove_calypso_sidebar_header' ) );
+	}
+
+	/**
+	 * Render action header
+	 */
+	public function render() {
+		?>
+		<div class="action-header">
+			<?php $this->back_button(); ?>
+			<div class="action-header__content">
+				<?php $this->site_icon(); ?>
+				<div class="action-header__details">
+					<?php $this->site_title(); ?>
+					<?php $this->breadcrumbs(); ?>
+				</div>
+			</div>
+			<div class="action-header__actions"></div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Back button
+	 */
+	public function back_button() {
+		?>
+		<a class="button action-header__ground-control-back" aria-label="<?php esc_html_e( 'Close Store', 'wc-calypso-bridge' ); ?>">
+			<svg class="gridicon gridicons-cross" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M18.36 19.78L12 13.41l-6.36 6.37-1.42-1.42L10.59 12 4.22 5.64l1.42-1.42L12 10.59l6.36-6.36 1.41 1.41L13.41 12l6.36 6.36z"></path></g></svg>
```

Comment:
Can these be updated once #166 lands?

Reply:
Definitely!  Once we get the 👍 on #166 I'll make sure these get replaced.